test/controllers/static_pages_controller_test.rb

```diff
@@ -1,15 +1,27 @@
 require 'test_helper'
 
 class StaticPagesControllerTest < ActionDispatch::IntegrationTest
+  test 'should get home' do
+    get root_path
+    assert_response :success
+    assert_select "title", "Ruby on Rails Tutorial Sample App"
+  end
+
   test 'should get help' do
-    get help_url
+    get help_path
     assert_response :success
     assert_select "title", "Help | Ruby on Rails Tutorial Sample App"
   end
 
   test 'should get about' do
-    get about_url
+    get about_path
     assert_response :success
     assert_select "title", "About | Ruby on Rails Tutorial Sample App"
   end
+
+  test 'should get contact' do
+    get contact_path
+    assert_response :success
+    assert_select "title", "Contact | Ruby on Rails Tutorial Sample App"
+  end
```

Comment:
[Q]
we have website logo and rails logo. Could you add more test to make sure both of them available?